src/Faker/Calculator/Iban.php

```diff
@@ -25,11 +25,10 @@ public static function checksum($iban)
     }
 
     /**
-     * @param string $match
-     *
+     * @param mixed $match
      * @return int
      */
-    private static function alphaToNumberCallback($match)
+    private static function alphaToNumberCallback($match): int
```

Comment:
According to the old php spec this should be string but according the tests its an array. Anyone who can confirm?